### ISSUE TITLE
Options not passed along to mongodb

### ISSUE THREAD

**Issue by tswaters** [balderdashy/sails-mongo#212]
In [./lib/connection.js around line 97](https://github.com/balderdashy/sails-mongo/blob/master/lib/connection.js#L97-L104) there is a "server_options" set on the options object that willl be passed to mongodb -- however, the driver is expecting a "server" object for these options.... The result is any configuration options passed to the adapter do not make their way into the driver, and mongodb defaults are used always.

I tried to hack it to be just "server" and this seems to work, however there is another problem -- the defaults that sails-mongo provides in .\lib\adapter.js actually make their way to the driver -- and a validation error gets raised -- keepAlive on socketOptions, specifically, needs to be a numeric and not a boolean.... looks like that was fixed in fb7a2d7 but that hasn't made it's way to npm yet.

Also it appears there is a PR for this, #187.

**Comment by sailsbot** [comment]
Thanks for posting, @tswaters.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by tswaters** [comment]
It should be noted this change is in master ans the 11.x branch -- when I wrote that the latest version was still 10.x I believe and while the fix was in master, it wasn't available in npm.  Update to use 11.x if you need to use server options passed to the driver.
